docs/en/reference/transactions-and-concurrency.rst

```diff
@@ -104,18 +104,11 @@ functionally equivalent to the previously shown code looks as follows:
         $em->persist($user);
     });
 
-.. warning::
-
-    For historical reasons, ``EntityManager#transactional($func)`` will return
-    ``true`` whenever the return value of ``$func`` is loosely false.
-    Some examples of this include ``array()``, ``"0"``, ``""``, ``0``, and
-    ``null``.
 
 The difference between ``Connection#transactional($func)`` and
 ``EntityManager#transactional($func)`` is that the latter
 abstraction flushes the ``EntityManager`` prior to transaction
-commit and rolls back the transaction when an
```

Comment:
Of course it is. I removed it because the original text said it was a difference from Connection::transactional() - implying that Connection::transactional() does not call the rollback - which is of course not the case.